docs/1_exist_data_model.md

```diff
@@ -44,6 +44,8 @@ for frame in video:
 
 A notebook demo can be found in [demo/inference_demo.ipynb](https://github.com/open-mmlab/mmdetection/blob/master/demo/inference_demo.ipynb).
 
+Note: `inference_detector`only supported single image
```

Comment:
Note: `inference_detector`only supported single-image inference for now.

Reply:
supports